examples/AspNetCore/ControllerSample/Controllers/SampleController.cs

```diff
@@ -83,6 +86,53 @@ public async Task<ActionResult<Account>> Deposit(Transaction transaction, [FromS
             await state.SaveAsync();
             return state.Value;
         }
+        
+        /// <summary>
+        /// Method for depositing to an account as specified in transaction. The topic is configured to receive
+        /// multiple messages at the same time, i.e. Bulk Subscribe.
+        /// </summary>
+        /// <param name="bulkMessage">Represents a bulk of messages received from the message bus.</param>
+        /// <param name="daprClient">State client to interact with Dapr runtime.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        ///  "pubsub", the first parameter into the Topic attribute, is name of the default pub/sub configured by the Dapr CLI.
+        [Topic("pubsub", "multideposit", "amountDeadLetterTopic", false)]
+        [TopicMetadata("bulkSubscribe", "true")]
+        [TopicMetadata("maxBulkSubCount", "10")]
+        [TopicMetadata("maxBulkSubAwaitDurationMs", "2000")]
```

Comment:
I like that we were able to reuse the metadata here. Is it possible to make a more convenient function for this? Or a `[BulkSubscribeTopicMetadata]` that flows through the same path? I know we originally didn't want a new annotation but I was just thinking if we follow the path of the metadata but provide a convenient annotation for it.

Reply:
Just one point here, bulk subscribe is no longer part of metadata, but it is its own field similar to route in subscriptions.

Reply:
Addressed